beacon/blockchain/payload.go

```diff
@@ -48,7 +48,35 @@ func (s *Service) forceSyncUponProcess(
 
 	// TODO: Verify if the slot number is correct here, I believe in current
 	// form it should be +1'd. Not a big deal until hardforks are in play though.
-	if err = s.localBuilder.SendForceHeadFCU(ctx, st, slot+1); err != nil {
+	slot++
+
+	lph, err := st.GetLatestExecutionPayloadHeader()
```

Comment:
this is the implementation that was in `localBuilder.SendForceHeadFCU`, without the `IsEnabled`